src/Fixer/ClassNotation/SingleTraitInsertPerStatementFixer.php

```diff
@@ -43,8 +43,8 @@ final class Example
 
     public function getPriority()
     {
-        // must be run before Braces and SpaceAfterSemicolonFixer
-        return 1;
+        // should be run before Braces, SpaceAfterSemicolonFixer
```

Comment:
Why changing `should` to `must`? It must run in that order for single run fixing, `should` suggest it's optional.

Reply:
Consistency: most other priority-comments use `should`

Reply:
@dmvdbrugge really? Can you point a fixer in `2.15` that is using should? ;)

Reply:
See the other files in this diff, most of them use "should" and comma-separation. There is another inconsistency using "be" or not but whatever.

I do find it interesting that *this* is the point you comment on after all this time 😄

Reply:
> See the other files in this diff

I did not ask about the diff, but the `2.15` branch ;)

Reply:
I meant of course the unchanged lines that can also be seen in the diff, which is based upon 2.15

If you meant to say that in the last 2 years something has changed (which for some reason is not visible in the diff), please just do so.

Reply:
I was hoping you find it yourself e.g. during doing rebase, but ok, this happened: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4758

Reply:
I'm currently not able to rebase (work...) but I can always read comments. I suspected something like that might've happened, but a simple "this has changed now" would've been a more effective way of re-opening the discussion, instead of challenging me on something that was correct when I wrote it (and according to the unchanged parts of the diff still is, blame github for that).

Thanks for the link, I will keep that in mind when I get around to rebasing this.